pkg/clusterconfig/clusterconfig.go

```diff
@@ -152,7 +152,20 @@ func GetAWSConfig(listers *regopclient.Listers) (*Config, error) {
 	return cfg, nil
 }
 
-func GetGCSConfig() (*Config, error) {
+func GetGCSConfig(listers *regopclient.Listers) (*Config, error) {
 	cfg := &Config{}
+
+	// Look for a user defined secret to get the GCS credentials from
+	sec, err := listers.Secrets.Get(imageregistryv1.ImageRegistryPrivateConfigurationUser)
+	if err != nil {
+		return nil, err
+	} else {
+		if v, ok := sec.Data["STORAGE_GCS_KEYFILE"]; ok {
```

Comment:
Add a comment here that GCS credentials are stored in a keyfile provided by Google. 

See also [1] and [2].

[1] https://cloud.google.com/storage/docs/authentication#service_accounts
[2] https://cloud.google.com/docs/authentication/production

Reply:
updated in next push